pkg/velero/velero.go

```diff
@@ -0,0 +1,32 @@
+package velero
```

Comment:
Can you add another task to verify copyright checks, something to the effect of https://github.com/openebs/maya/blob/master/buildscripts/travis-build.sh#L41

Reply:
done. I'll address it in separate PR.